tests/conftest.py

```diff
@@ -1,3 +1,31 @@
+import pytest
+
+
 def pytest_addoption(parser):
 
     parser.addoption("--ya-api-key", type=str, help="instance api key", dest="yaApiKey")
+
+
+@pytest.fixture
+def engine_starts_without_yagna(monkeypatch):
```

Comment:
I'd call this `yagna_mock` or `yagna_fixture` since that's what the fixture gives you ... 

`engine_starts_without_yagna` sounds extremely awkward imo...